depends/funcs.mk

```diff
@@ -77,7 +77,7 @@ $(1)_download_path_fixed=$(subst :,\:,$$($(1)_download_path))
 
 #default commands
```

Comment:
In commit "depends: tar: Always extract as yourself" (bef1df52301e57ba53ed38cbc038529f76c7ee5b)

Could mention purpose of `--no-same-owner` in a comment here (tar will try to set ownership when running as uid 0 and may not succeed), since it's not so obvious